app/controllers/forms_controller.rb

```diff
@@ -100,7 +101,7 @@ def odk_manifest
       @ifa = ItemsetsFormAttachment.new(form: @form)
       @ifa.ensure_generated
     end
-    render_openrosa
+    render
```

Comment:
Probably, I think that was the result of a find/replace which would've hit places where they may not have been explicit as well, I'll clean that up.